src/__tests__/fixtures/bundle-bug-9567.json

```diff
@@ -0,0 +1,83 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "bug 9567"
+  },
+  "components": {
+    "responses": {
+      "representative": {
+        "content": {
+          "application/json": {
+            "schema": {
+              "title": "representative_response",
+              "allOf": [
+                {
+                  "title": "user_reference",
+                  "type": "object",
+                  "properties": {
+                    "user_id": {
+                      "type": "string",
+                      "format": "uuid"
+                    }
+                  },
+                  "required": ["user_id"]
+                }
+              ]
+            }
+          }
+        }
+      }
+    },
+    "schemas": {
+      "webhook_representative": {
+        "allOf": [
+          {
+            "type": "object",
+            "properties": {
+              "data": {
+                "$ref": "#/components/responses/representative/content/application~1json/schema"
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "__target__": {
+    "id": "?http-operation-id?",
+    "iid": "get-users-user_id",
+    "method": "get",
+    "path": "/users/{user_id}",
+    "responses": [
+      {
+        "code": "200",
+        "headers": [],
+        "contents": [
+          {
+            "mediaType": "application/json",
+            "schema": {
+              "title": "user_response",
+              "oneOf": [
+                {
+                  "title": "user_natural_response",
+                  "allOf": [
+                    {
+                      "title": "user_base_response",
+                      "allOf": [
+                        {
+                          "$ref": "#/components/schemas/webhook_representative/allOf/0/properties/data/allOf/0"
```

Comment:
what's not valid about it?

Reply:
I ask because that ref works as far as I know.

Reply:
yeah, it works because we tend to support it, but it's not valid from the spec's point of view.
I linked our internal ticket we use for tracking that behavior and there's one additional ticket linked there too which sheds more light on the issue.